internal/command/command.go

```diff
@@ -512,18 +512,33 @@ var errRequireAppName = fmt.Errorf("we couldn't find a fly.toml nor an app speci
 // application name via command line arguments, the environment or an application
 // config file (fly.toml). It embeds LoadAppConfigIfPresent.
 func RequireAppName(ctx context.Context) (context.Context, error) {
-	ctx, err := LoadAppConfigIfPresent(ctx)
+	platform := appv2.AppsV1Platform
+	newCtx, err := LoadAppConfigIfPresent(ctx)
```

Comment:
This function basically parses fly.toml into a `map[string]any` which covers pretty much everything. What would make this fail for a v2 app?

Reply:
What wasn't working before the changes to this file?

Reply:
I think this was while I was trying to fix a bug with loading the config for apps v2, it doesn't actually change any behavior but it's also unnecesssary. I'll revert it for simplicity.